src/main/scala/AI.scala

```diff
@@ -0,0 +1,7 @@
+
+class AI(override val name: String = "RPS bot") extends Player {
+  override def play: RPS.Play = {
```

Comment:
why not 

override def play: RPS.Play = RPS(scala.util.Random.nextInt(RPS.maxId))